src/processor/Package.ts

```diff
@@ -39,11 +42,20 @@ export class Package {
       this.valueSets.push(resource);
     } else if (resource instanceof ExportableCodeSystem) {
       this.codeSystems.push(resource);
+    } else if (resource instanceof ExportableConfiguration) {
+      if (this.configuration) {
+        logger.warning(
+          `Multiple implementation guide resources found in input folder. Skipping implementation guide with ${
+            resource.config.id ? `id ${resource.config.id}` : 'unknown id'
```

Comment:
Here as a backup, instead of `unknown id` we could consider outputting the `canonical`, since we know that will be defined right?

Reply:
I wasn't seeing a test for the logic of skipping previous IGs here. Could you add a test for that?